src/app/state/account/saga.ts

```diff
@@ -17,6 +17,17 @@ import { selectAccountAddress } from './selectors'
 export function* fetchAccount(action: PayloadAction<string>) {
   const address = action.payload
 
+  if (!isValidAddress(address)) {
+    yield* put(
```

Comment:
I'd remove if we don't need

Reply:
We don't strictly need it at this point, so I removed